homeassistant/components/iss/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "description": "Do you want to configure the Internation Space Station?",
```

Comment:
We do not need to ask the user this, that is already implied, as they selected adding this integration

Please remove this translation.